src/lrs_annotation/snps_indels_annotation_stages.py

```diff
@@ -328,6 +328,11 @@ def queue_jobs(self, dataset: targets.Dataset, inputs: stage.StageInput) -> stag
             dataset (Dataset): SGIDs specific to this dataset/project
             inputs ():
         """
+        eligible_datasets = config_retrieve(['workflow', 'write_mt_for_datasets'], default=[])
+        if dataset.name not in eligible_datasets:
```

Comment:
Does this mean you always need to specify eligible_datasets and is that what you want?
I would have assumed you would do something like:
```if eligible_datasets and dataset.name not in eligible_datasets:```

Or, more likely, I am missing something...